### DISCUSSION TITLE
Where to put repositories with windows dependencies?

### DISCUSSION THREAD

**Discussion by oleg68** [GrandOrgue/grandorgue#722] [Developers]
For build GO for windows we need some dependencies. Earlier these dependencies were got from OpenSuse repositories made by Martin Koegler.

Because GitHub runs builds in ubuntu runners, we need dependencies packaged as .deb packages #28. I have already created two repositories that release such packages: https://github.com/oleg68/JackCross and https://github.com/oleg68/WxWidgetsCross.

@larspalo @rousseldenis  May I put them into https://github.com/GrandOrgue ?

**Comment by larspalo** [comment]
@oleg68 As repositories under GrandOrgue organization, right?

**Reply by oleg68** [reply]
Yes, I'd put it there.

**Reply by larspalo** [reply]
Ok, as they are dependencies it makes sense that they are under the organization!

**Comment by oleg68** [comment]
@rousseldenis Do you argee with moving my repositories 
- https://github.com/oleg68/Mingw64LibGnuRx
- https://github.com/oleg68/PortaudioMingw
- https://github.com/oleg68/JackCross
- https://github.com/oleg68/WxWidgetsCross

into GrandOrgue organisations?

**Reply by rousseldenis** [reply]
Yes, no problem. One question, are these should be in different repos or could we imagine having them in the same one ?

**Reply by oleg68** [reply]
Of cause, different. They are needed only for building GO for windows

**Reply by rousseldenis** [reply]
Ok, no problem. I wanted just to know if we can merge them as they do quite the same (GitHub actions and build scripts)

**Comment by oleg68** [comment]
I moved four repositories to GrandOrgue
